class7-week3/models/todo.js

```diff
@@ -74,6 +74,62 @@ class Todo {
       })
     })
   }
+    reset() {
+    this.load((error, todos) => {
+      if (error) { callback(error); return }
+
+      todos = []
+
+      this.save(todos, error => {
+        if (error) { callback(error); return }
+      })
+    })
+        
+    }
+    markAsDone(id, done, callback) {
+    this.load((error, todos) => {
+      if (error) { callback(error); return }
+
+      const todo = todos.find(t => t.id === id)
+      if (todo == null) {
+        const error = new Error(`Todo with ID ${id} does not exist`)
+        error.name = 'NotFound'
+
+        callback(error)
+        return
+      }
+
+      todo.done = 'True'
+
+      this.save(todos, error => {
+        if (error) { callback(error); return }
+
+        callback(null, todo)
+      })
+    })
+  }
+    markAsNotDone(id, done, callback) {
```

Comment:
Also here - you're bringing in `done` but never using it.

Hint: you don't need it.